File: Java/Promedio.java
import java.io.*;

public class Promedio {
    static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

    public static void main(String[] args) throws Exception {

        float[] calificaciones = new float[7];
        float suma = 0;

        for (int i = 0; i < calificaciones.length; i++) {
            System.out.print("Digite la calificacion " + (i + 1) + ": ");
            calificaciones[i] = Float.parseFloat(br.readLine());
        }

        for (int i = 0; i < calificaciones.length; i++) {
            suma += calificaciones[i];
        }

        float promedio = suma / 7;
        String promedioFormateado = String.format("%.3f", promedio);
        System.out.println("El promedio es: " + promedioFormateado);
    }
}

// https://omegaup.com/arena/problem/Promedio-Calificaciones-/

File: Python/Promedio.py
def main():
    calificaciones = [0.0] * 7
    suma = 0.0

    for i in range(len(calificaciones)):
        calificaciones[i] = float(input(f"Digite la calificacion {i + 1}: "))

    for calificacion in calificaciones:
        suma += calificacion

    promedio = suma / 7
    promedio_formateado = "{:.1f}".format(promedio)
    print(f"El promedio es: {promedio_formateado}")

main()

# https://omegaup.com/arena/problem/Promedio-Calificaciones-/